rudder-web/src/test/resources/api/api_v2.yml

```diff
@@ -0,0 +1,15 @@
+---
+description: Simple test of api/status - just "ok"
+method: GET
+url: /api/status
+response: 
+  type: text #optionnal, by default json
+  code: 200
+  content: 'OK'
+---
+method: GET
+url: /api/techniqueLibrary/reload
+response:
+  type: plop
```

Comment:
Yes, I wanted to test parsing error. 
For now, the behaviour is to log:
```
[2016-11-15 11:33:32] ERROR com.normation.rudder.web.rest.TestRestFromFileDef - I wasn't able to run the 2th tests in file api/api_v1.yml <- The YAML document is empty
[2016-11-15 11:33:33] ERROR com.normation.rudder.web.rest.TestRestFromFileDef - I wasn't able to run the 1th tests in file api/api_v2.yml <- Unrecognized response type: 'plop'. Use 'json' or 'text'
```
And the test is not run. Would you find better to also fail the test? Effectively, it could be easier to correct all problems when we will change the format.